test/Microsoft.Sbom.Api.Tests/Workflows/SbomParserBasedValidationWorkflowTests.cs

```diff
@@ -390,4 +392,147 @@ public async Task SbomParserBasedValidationWorkflowTests_ReturnsSuccessAndValida
         fileSystemMock.VerifyAll();
         osUtilsMock.VerifyAll();
     }
+
+    [TestMethod]
+    public async Task SbomParserBasedValidationWorkflowTests_ReturnsNTIAValidationFailures_Succeeds()
+    {
+        var manifestInfo = Constants.SPDX30ManifestInfo;
```

Comment:
Not in this PR, but we _desperately_ need to rethink these tests. No test should take this much setup.

Reply:
Agreed I'll create an issue/work item for it